pkgs/applications/misc/ipmicfg/default.nix

```
@@ -22,14 +23,15 @@ stdenv.mkDerivation rec {
     ln -s "$out/opt/ipmicfg/IPMICFG-Linux.x86_64" "$out/bin/ipmicfg"
```

For some reason i cannot run the binary. Has this been an issue for long?

```suggestion
    chmod +x "$out/opt/ipmicfg/IPMICFG-Linux.x86_64"
    ln -s "$out/opt/ipmicfg/IPMICFG-Linux.x86_64" "$out/bin/ipmicfg"
```